ecs_fargate/metadata.csv

```diff
@@ -0,0 +1,24 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+ecs.fargate.io.ops.write,rate,,,,# write operations to the disk,0,fargate,io write count
+ecs.fargate.io.bytes.write,rate,,byte,,# bytes written to the disk,0,fargate,io write
+ecs.fargate.io.ops.read,rate,,,,# read operation on the disk,0,fargate,io read count
+ecs.fargate.io.bytes.read,rate,,byte,,# bytes read on the disk,0,fargate,io read
+ecs.fargate.cpu.user,rate,,nanosecond,,Cpu time spent in ns,0,fargate,cpu user
```

Comment:
Shouldn't it be a `gauge` like `docker.cpu.usage` and friends?
@hkaj

Reply:
let's do what we do in the docker check since its the same metrics